src/main/java/fruitshop/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,18 @@
+package fruitshop.strategy;
+
+import fruitshop.db.Storage;
+import fruitshop.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void handler(FruitTransaction fruitTransaction) {
+        Storage storage = new Storage();
```

Comment:
Creating a new instance of Storage inside the handler method is not recommended. Instead, use the existing Storage instance. If you need to create a Storage object, it should be passed as a dependency through the constructor, ensuring that you're working with the same storage throughout the application.